File: 229-majority-element-ii/majority-element-ii.cpp
class Solution {
public:
    vector<int> majorityElement(vector<int>& nums) {
        int n = nums.size();
        unordered_map<int,int> mpp;
        for(int ele: nums){
            mpp[ele]++;
        }
        vector<int> ans;
        for(auto ele: mpp){
            if (ele.second > n/3){
                ans.push_back(ele.first);
            }
        }
        return ans;
        
    }
};

File: 229-majority-element-ii/majority-element-ii.py
class Solution:
    def majorityElement(self, nums: List[int]) -> List[int]:
        n = len(nums)
        threshold = n // 3
        result = []
        
        for num in set(nums): 
            count = nums.count(num) 
            if count > threshold:
                result.append(num)
        
        return result
